uswds/uswds-site
# USWDS - CircleCI: Restore gemfile .lock

## Description

**Resolve cache errors in circleci.** Restore `gemfile.lock` to avoid missing file error in builds. Closes #1947.

## Additional information

Reverts to a previous setup where we versioned `gemfile.lock`. Follows this CI guidance from [Jekyll docs](https://jekyllrb.com/docs/continuous-integration/circleci/). 

⚠️ `gemfile.lock` was removed to avoid local build errors since some developers are on M1 macs. We'll to find an alternative solution to that.

## How to test

There should be no cache errors in circleci builds.

![image](https://user-images.githubusercontent.com/3385219/208169666-e24108ca-06fe-4ed5-bf42-d26d1bd885c1.png)


---

Before you hit Submit, make sure you’ve done whichever of these applies to you:
